packages/marketplace/src/logic/Utils.ts

```diff
@@ -19,6 +19,7 @@ export const getLocalStorageDataFromKey = (key: string, fallback?: unknown) => {
       return data;
     }
   } else {
+    console.log(fallback);
```

Comment:
You shouldn't need to `console.log` this if we were going to merge this to `main`?

Reply:
Forgot to remove this lol